docs/openapi/components/schemas/credentials/MillTestReportCredential.yml

```diff
@@ -4,6 +4,7 @@ $linkedData:
 title: Mill Test Report Credential
 tags:
   - Steel
+  - TechDemo2023
```

Comment:
Add a tag like this.

Reply:
This should probably be a catair tag not a techdemo2023 tag